main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
+more details. 
+
+## Simoleons
+An imaginary currency Agoric documentation uses in examples.
+
+## Terms
+Contract instances have associated terms, gotten via `E(zoe).getTerms(instance)`,
+which include the instance's associated [issuers](#issuer), [brands](#brand), and any custom terms. For 
+example, you might have a general auction contract. When someone instantiates it,
+they provide terms applicable only to that instance. For example, for some instances of 
+the auction, they want the minimum bid set at $1000. At other instances, they'd like
+it set at $10. They can specify the instance's minimum bid amount in its terms.
+
 ## Value
 
 Values are the part of an [amount](#amounts) that describe the value of something
 that can be owned or shared: How much, how many, or a description of a unique asset, such as
-Pixel(3,2), $3, or ‘Right to occupy on Tuesdays’. [Fungible](#fungible) values are usually 
+Pixel(3,2), $3, or 'Seat J12 for the show September 27th at 9:00pm'.
+[Fungible](#fungible) values are usually 
 represented by natural numbers. Other values may be represented as strings naming a particular
 right, or an arbitrary object representing the rights at issue. The latter two examples 
 are usually [non-fungible](#non-fungible) assets. Values must be [Comparable](#comparable).
 
-See the [ERTP Guide's Value section](/ertp/guide/amounts.md#values) for more information.
+For more information, see the [ERTP Guide's Value section](/ertp/guide/amounts.md#values).
+
+## Vat 
+A vat is a unit of isolation. To paraphrase the Las Vegas advertising slogan, what happens in a vat stays in that vat.
+Objects and functions in a JavaScript vat can communicate synchronously with one another. Vats and their contents can
+communicate with other vats and their objects and functions, but have to manage asynchronous messages and responses.
+
+For more information, see the [Vat section in the Distributed JS Programming Guide](/distributed-programming.md#vats)
+
+## Wallet
+
+The overall place a party keeps their assets of all brands. For example, your wallet might contain 5 Quatloos
+[purses](#purse), 8 Moola purses, and 2 Simoleons purses. You can also keep [Issuers](#issuer) in a 
+wallet. [Offers](#offer) are associated with the wallets 
+from which their associated payments come from. See the [Wallet API](/wallet-api.md).
+
+## ZCF
+*ZCF (Zoe Contract Facet)* is the [facet](#facet) of Zoe exposed to contract code. The Zoe 
+Contract Facet methods can be called synchronously by contract code. 
+
+See the [ZCF API](/zoe/api/zoe-contract-facet.md).
 
-## Vat
+## ZCFMint
 
-A vat is a *unit of synchrony*. This means that within a JavaScript vat, objects and functions can communicate with one another synchronously.
+ZCFMints are Zoe mints. Similar to ERTP mints, they have one-to-one relationships
+with an [issuer](#issuer) and its associated [brand](#brand). A ZCFMint can mint assets that get associated with a [seat](#seat)
```

Comment:
See the above language under mint for fixing `Zoe mint` and `similar to ERTP mints`.